lib/modules/media/media_models.dart

```diff
@@ -380,10 +382,33 @@ class MediaInfo {
       }
     }
 
+    if (map['externalLinks'] != null) {
+      for (final link in map['externalLinks']) {
+        model.externalLinks.add(ExternalLink(link));
+      }
+    }
+
     return model;
   }
 }
 
+class ExternalLink {
+  ExternalLink._({required this.url, required this.site, required this.color});
+
+  final String url;
+  final String site;
+  final Color? color;
+
+  factory ExternalLink(Map<String, dynamic> map) => ExternalLink._(
```

Comment:
I meant that you move the fields after the factory too 😅

Reply:
Otherwise, it's all great.

Reply:
Oh my bad 🤦🏾 

I've made the change